src/main/resources/assets/paths/studyById.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  summary: Get Study by Id
+  description: Finds Study and Properties of the Study specified by id.
+  parameters:
+    - name: id
+      in: path
+      description: ID of the Study
+      required: true
+      schema:
+        type: integer
+  tags:
+    - Study
```

Comment:
This is a new category, which makes sense. We should probably put it in both, `Dataset` and `Study`. Also, since we're adding `Study` here, it would make sense to tag `/api/dataset/studyNames` as well.